docker-compose.yml

```diff
@@ -40,6 +40,10 @@ services:
         - postgresql
         - mailcatcher
         - redis
+    # FIXME: temporary configuration to connect AC and desk
```

Comment:
this could remain in the project and doesn't have any side effects even if you rarely need it?